acra/acra-in-depth/architecture/dbs.md

```diff
@@ -4,31 +4,61 @@ bookCollapseSection: true
 weight: 4
 ---
 
-**TODO: Expand**
+# Key storage
 
-# Key storage 
-
-Is used to store encrypted keys and tokens. 
+Is used to store encrypted keys and tokens.
 
 ## How it works
 
+A dedicated database (we currently support Redis for this purpose) is brought up.
```

Comment:
better to use `kv storage` instead of database. because we rely on KV API on base level and try to not depends on more complicated API that databases may provide to be as simple as possible to extend.